client/my-sites/themes/themes-magic-search-card/index.jsx

```diff
@@ -296,7 +296,7 @@ class ThemesMagicSearchCard extends React.Component {
 							</div>
 						}
 						{
-							( isPremiumThemesEnabled && ( config.isEnabled( 'jetpack/pijp' ) || ! isJetpack ) ) &&
+							( isPremiumThemesEnabled && ! isJetpack ) &&
```

Comment:
Same as https://github.com/Automattic/wp-calypso/pull/15863/files#r125625605? We want to show the `SegmentedControl` now regardless, including JP sites.